README.md

```diff
@@ -86,35 +86,36 @@ $ kubectl run receiver-mock \
 
 ## K8S Template generator
 
+Before generating the configuration we recommend to prepare `values.yaml` file where you will store all your configuration.
+Alternatively you can replace the file with `--set property=value` arguments according to [helm documentation](https://helm.sh/docs/intro/using_helm/).
+
 Docker based:
 
 ```bash
-docker run --rm sumologic/kubernetes-tools \
+docker run \
+  -v $(pwd)/values.yaml:/values.yaml \
+  --rm sumologic/kubernetes-tools \
   template \
   --namespace '<NAMESPACE>' \
   --name-template 'collection' \
-  --set sumologic.accessId='<ACCESS_KEY>' \
-  --set sumologic.accessKey='<ACCESS_ID>' \
-  --set sumologic.collectorName='<COLLECTOR_NAME>' \
-  --set sumologic.clusterName='<CLUSTER_NAME>' \
+  -f values.yaml \
   | tee sumologic.yaml
 ```
 
 Kubernetes based:
 
-```
-kubectl run tools \
-  -it --rm \
-  --restart=Never \
-  --image sumologic/kubernetes-tools -- \
+```bash
+kubectl create configmap sumologic-values --from-file=values.yaml
+curl https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-tools/master/src/k8s/tools-pod.yaml -s | kubectl apply -f -
```

Comment:
Should we link to master or should we have a proper branch so that as we make changes the master link does not get anything?